pkg/executor/executortype/container/containermgr.go

```diff
@@ -781,3 +782,34 @@ func getDeploymentObj(kubeobjs []apiv1.ObjectReference) *apiv1.ObjectReference {
 	}
 	return nil
 }
+
+func getObjectReaperInterval(logger *zap.Logger, defaultReaperInterval int) int {
```

Comment:
Rather than defining this function at all three executor types, I am just thinking we can write this in better fashion and avoid code duplication.


```
func getObjectReaperInterval(logger *zap.Logger, executorType  ExecutorType, defaultReaperInterval int) int
```

We can pass these parameters at the time of function call. Defined in `pkg/apis/core/v1`

```
const (
	ExecutorTypePoolmgr   ExecutorType = "poolmgr"
	ExecutorTypeNewdeploy ExecutorType = "newdeploy"
	ExecutorTypeContainer ExecutorType = "container"
)
```

Variable names for particular executor types can be constructed at runtime.

https://go.dev/play/p/XDAOUeQU5kM
 So this can generalize code. 

Tomorrow if we think of defining variables at each executor level we can use the same approach.

Reply:
Absolutely agree. I was confused by the fact that the ISTIO settings are duplicated in each separately.
What do you think about putting my method in pkg/utils/utils.go?

Reply:
probably `pkg/executor/util/util.go` would be good place for this IMO. Since function is executor specific.

Reply:
@Tri0L 
You will need to make changes in helm charts as well, so user can set the values for the same while installing fission.
POOLMGR_OBJECT_REAPER_INTERVAL
NEWDEPLOY_OBJECT_REAPER_INTERVAL
CONTAINER_OBJECT_REAPER_INTERVAL
OBJECT_REAPER_INTERVAL

Reply:
@shubham-bansal96 Yes, I will add them. I just wanted to make sure that my edits basically make sense and can be accepted.

Reply:
@Tri0L your changes looks good, Please go ahead with helm chart changes.

Reply:
I added helm chart settings, and tests